Rpackage/DESCRIPTION

```diff
@@ -6,6 +6,6 @@ License: GPL-3
 Title: Max Margin Interval Trees
 Description: Fast O(P N log N) algorithm for learning a regression tree
  with interval censored output data.
-Depends: partykit, penaltyLearning, assertthat, doParallel, parallel, foreach
+Depends: partykit, penaltyLearning, assertthat, parallel, doParallel, future.apply
```

Comment:
remove doParallel since that is not used anymore right?

Change Depends to Imports please! (and update the NAMESPACE accordingly)

Depends means that when you do library(mmit) it will do library(pkg) for each of the Depends packages. (which attaches each Depends package, making all of their exported functions available for the user)

Typically Imports is preferred because when the user types library(mmit) they usually want access to mmit functions but probably not all of partkit, penaltyLearning, assertthat, etc functions.